controllers/storagecluster/provider_server.go

```diff
@@ -249,6 +252,9 @@ func GetProviderAPIServerService(instance *ocsv1.StorageCluster) *corev1.Service
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      ocsProviderServerName,
 			Namespace: instance.Namespace,
+			Annotations: map[string]string{
+				serviceCertAnnotation: ocsProviderServerName + "-cert",
```

Comment:
how about using a new global variable instead of  `ocsProviderServerName + "-cert"`

Reply:
done